### PR TITLE
Feat individual deploy

### PR DESCRIPTION
**Pull request by notanmay**

### CODE REVIEW

File: lib/deploy/deploy.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+
+module.exports = {
+  /*
+   * Will get stack resources and create object and append in obj
+   * with key as ARN Name and value as ARN Value
```
Comment (KillDozerX2) [inline lib/deploy/deploy.js]: Please add the description that you explained to me.

File: lib/deploy/deploy.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      return BbPromise.resolve();
+    });
+  },
+  replaceAllStatesARNInDefinition(definition, value) {
+    function iterateStates(obj) {
+      const data = {};
+      data.States = {};
+      for (const key in obj) {
+        if (key === 'States') {
+          const subStates = obj[key];
```
Comment (KillDozerX2) [inline lib/deploy/deploy.js]: Please use [Object.entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries)

Comment (jappurohit041) [reply lib/deploy/deploy.js]: Used and modified logic along with simplifying calling.

File: lib/deploy/deploy.js (hunk 1 above)
Comment (jappurohit041) [reply lib/deploy/deploy.js]: Added detailed description

**Review by KillDozerX2 — APPROVED** [review]

**Pull request merged**